Tests/OpusTests/OpusDecoderTests.swift

```diff
@@ -15,4 +15,24 @@ final class OpusDecoderTests: XCTestCase {
 			}
 		}
 	}
+
+
+	func test_CorruptedOpusData_ThrowsInvalidPacketError() throws {
```

Comment:
I'd write this without the underscores but it's a minor nit. Either way is fine.

Reply:
sure, no problem, done!